src/App.java

```diff
@@ -1,33 +1,93 @@
 import java.util.Scanner;
 
 public class App {
-    
+
     /**
      * This method is the primary driver method for your program.
      * It should primarily call other methods to do its work.
      * Do NOT edit this method until after you have fully satisfied the reqs for a 5
      */
+    public static String Blue = "\u001B[34m";
+    public static String Reset = "\u001B[0m";
+    public static String Red = "\u001B[31m";
+    public static String Green = "\u001B[32m";
+    public static String Yellow = "\u001B[33m";
+    public static String Purple = "\u001B[35m";
+    public static String Cyan = "\u001B[36m";
 
+    // https://www.geeksforgeeks.org/how-to-print-colored-text-in-java-console/
```

Comment:
This link does not seem to match this code directly. More descriptive comment on how you learned to apply what you figured out from that link would have gone a long way to helping me understand where this code came from